### ISSUE TITLE
Add minimal numeric keyboard

### ISSUE THREAD

**Issue by HolgerJeromin** [westnordost/StreetComplete#336]
Perhaps overkill, but the numeric quests would benefit from a small row of numbers to click. 

There are other keyboards available, but they are not the optimal for every one.
Here an example for a small solution ( http://fivetiles.com/ ) which has no number layout:

![rps20170616_000905_817](https://user-images.githubusercontent.com/2410353/27203846-0c9945a2-5228-11e7-835e-7fa28083e3d2.jpg)

**Comment by westnordost** [comment]
I tried that, for the housenumber quest.

Keyboards in Android are a separate application. The app has no control over which keyboards the user uses and also not control over the keyboard itself. The app can only say which kind of input something is (number, text etc) and then has to hope that the installed keyboard does something meaningful.

There is the option to "fake" a keyboard which is actually just a view, but it is quite bad I can tell you, I tried it. It never looks native.

Sorry, this is not worth the effort.
